### DISCUSSION TITLE
Add option for "x:1" aspect ratio

### DISCUSSION THREAD

**Discussion by taep96** [ppy/osu#13193] [Ideas]
![image](https://user-images.githubusercontent.com/64481039/114549521-65495e80-9c61-11eb-85f3-f297cf6f4c0c.png)
Add a toggle to display aspect ratio like this.

**Comment by LittleEndu** [comment]
If there's enough room, wouldn't displaying both be a better solution

**Comment by peppy** [comment]
Are we really looking to display aspect ratios in a non-standard format just because some external application does? I'm not sure I can agree with this. Especially since we will support inputting the exact mm values which will result in the correct aspect ratio anyway.

**Comment by taep96** [comment]
Just add it for better UX?
